assets/js/guides.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function () {
+
+  //Dismiss cta on guides page
+  $('.dismiss-cta').on('click', (e) => {
+    e.preventDefault();
+    $('.guides-cta-card').hide();
+  })
+
+  //Validate email
+  let validateEmail = (email) => {
+    let re = /^([a-zA-Z0-9_.-])+@(([a-zA-Z0-9-])+.)+([a-zA-Z0-9]{2,4})+$/;
+    return re.test(email);
```

Comment:
Email addresses are more complicated than this Regex. The only hard and fast rule is it contains an `@`. Instead of this, use an `input type="email"`, and let it handle validation.

Reply:
(this is another fix that could go in a new newsletter-only PR)